src/backend/mod.rs

```diff
@@ -92,7 +93,7 @@ impl Backend {
                         let txs: Vec<Transaction> = blocks
                             .iter()
                             .skip(skip)
-                            .take(limit)
+                            .take(if limit == 0 {usize::MAX} else {limit} )
```

Comment:
Should 'take = 0' really be considered to mean that we want to take the entire list?